pyta.py

```diff
@@ -36,7 +48,13 @@ def check(module_name):
                                 'checkers/IO_Function_checker'])
     linter.read_config_file()
     linter.load_config_file()
-    linter.check([spec.origin])
+
+    # When module is not found, raise exception.
+    try:
+        linter.check([spec.origin])
+    except AttributeError:
+        print ("The Module '{}' could not be found in the given "
```

Comment:
I'm not sure what @nigef is suggesting the message should actually be.

@hayleyZHL the phrase "given location" is unclear. For now you can simply end the sentence after `found`.